tcms/rpc/api/forms/testrun.py

```diff
@@ -21,40 +23,51 @@ def assign_plan(self, plan_id):
         self.populate(self.fields['plan'].queryset.first().product_id)
 
 
-class UpdateForm(NewForm):
-    plan = forms.ModelChoiceField(
-        queryset=TestPlan.objects.all(),
-        required=False,
-    )
-    summary = forms.CharField(
-        required=False
-    )
-    manager = forms.ModelChoiceField(
-        queryset=User.objects.all(), required=False
-    )
-    product = forms.ModelChoiceField(
-        queryset=Product.objects.all(),
-        empty_label=None, required=False
-    )
-    product_version = forms.ModelChoiceField(
-        queryset=Version.objects.none(),
-        empty_label=None, required=False
-    )
-    build = forms.ModelChoiceField(
-        queryset=Build.objects.all(),
-        required=False
+class UpdateForm(forms.ModelForm):
+
+    manager = UserField()
+    default_tester = UserField(required=False)
+
+    start_date = forms.DateTimeField(
+        input_formats=['%Y-%m-%d', '%Y-%m-%d %H:%M:%S'],
```

Comment:
Can you move these into a module level variable. It doesn't look like there is a common constant with these formats anywhere in Python or Django. The only thing I was able to find is this comment:
# See "Date Time String Format" in the ECMA-262 specification.